.gitignore

```diff
@@ -24,3 +25,4 @@ src/memory_service/fam_memory_service_rpc.grpc.pb.h
 src/memory_service/fam_memory_service_rpc.pb.h
 src/metadata_service/fam_metadata_rpc.grpc.pb.h
 src/metadata_service/fam_metadata_rpc.pb.h
+src/allocator/.nfs0000001186a6003800000e5e
```

Comment:
Not sure if this is needed. "src/allocator/.nfs0000001186a6003800000e5e". Isnt this a temporary file.